include/zephyr/bluetooth/classic/spp.h

```diff
@@ -0,0 +1,137 @@
+/** @file
+ *  @brief Bluetooth SPP Protocol handling.
+ */
+
+/*
+ * Copyright 2025 Xiaomi Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SPP_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SPP_H_
+
+/**
+ * @brief SPP
+ * @defgroup SPP
+ * @ingroup bluetooth
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/bluetooth/buf.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/uuid.h>
+
+#include <zephyr/bluetooth/classic/rfcomm.h>
+
+struct bt_spp {

```

Comment:
This struct is missing documentation. It should also be made clear which fields are internal and which the application is allowed to access directly.